heft-plugins/heft-typescript-plugin/src/TranspilerWorker.ts

```diff
@@ -0,0 +1,83 @@
+import { parentPort, workerData } from 'node:worker_threads';
+
+import * as TTypescript from 'typescript';
+import type {
+  ITranspilationRequestMessage,
+  ITranspilationResponseMessage,
+  ITypescriptWorkerData
+} from './types';
+import type { ExtendedTypeScript } from './internalTypings/TypeScriptInternals';
+import { configureProgramForMultiEmit } from './configureProgramForMultiEmit';
+
+const typedWorkerData: ITypescriptWorkerData = workerData;
+
+const ts: ExtendedTypeScript = require(typedWorkerData.typeScriptToolPath);
```

Comment:
Is this how we're grabbing TypeScript elsewhere?

Reply:
Yep.